### PR TITLE
Update Optimizer docs & fix SGD+Mom WD comparison

### PR DESCRIPTION
**Pull request by warner-benjamin**
Update Optimizer docs with additional clarity relative to PyTorch optimizers.

Fix the SGD with Momentum Weight Decay comparison test/example.

Move `OptimWrapper` usage example to right after the `OptimWrapper` definition.

### CODE REVIEW

File: nbs/12_optimizer.ipynb [hunk 1]
```diff
@@ -243,7 +243,9 @@
    "source": [
```
Comment (jph00) [inline nbs/12_optimizer.ipynb:243]: <p>I think this addition could be a bit distracting/confusing. Especially with the "important" at the start. For most people I don't think it's important at all! Generally speaking I don't think you'd want wd applied to normalization bias terms, would you?</p><p> </p><p>If there are situations where someone wants to make a fastai optimizer work just like pytorch's, how about we instead create a little section in this notebook called "differences to pytorch optimizers", or something like that. That way, we can remove these notes scattered through the notebook, and have all that info in one place.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/fastai/fastai/pull/3733/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='13'/>
